docs/pages/architecture/authentication.mdx

```diff
@@ -1,230 +1,167 @@
+desktops and SSH certificates for OpenSSH-compatible servers.
 
-One can think of an SSH certificate as a "permit" issued and time-stamped by a
-trusted authority. In this case, the authority is the Auth Server's Certificate
-Authority. A certificate contains four important pieces of data:
+### Why certificates?
 
-1. List of principals (identities) this certificate belongs to.
-2. Signature of the certificate authority who issued it.
-3. The expiration date, also known as "time-to-live" or simply TTL.
-4. Additional data, such as the node role, is stored as a certificate extension.
-
-## Authentication in Teleport
-
-Teleport uses SSH certificates to authenticate nodes and users within a cluster.
-
-Two CAs are used inside the Auth Server because nodes and users each need their own certificates. {/* TODO: Why? */}
+- Certificates are tied to user or service identity. Any connection and action can be traced back to a user
+or a service.
+- Short-lived certificates automatically expire, there is no need to revoke them.
+- Certificates solve trust on first use (TOFU) problems. In a Teleport cluster, all servers have
+identities and certificates of their own. They will not allow a connection if a client certificate is signed by an
+untrusted certificate authority.
+- Certificates enable mutually authenticated channel - mTLS, mTLS mitigates a wide range of attacks - spoofing, on-path attacks, credential stuffing and others.
+- Certificates work better for large-scale deployments. Each server or service just needs to validate
+if the certificate has been signed with a valid certificate authority, and does not need to copy user
+credentials over to every service.
 
-- The **Node CA** issues certificates which identify a node (i.e. host, server,
-  computer). These certificates are used to add new nodes to a cluster and identify connections coming from the node.
-- The **User CA** issues certificates which identify a User. These certificates are used to authenticate users when they try to connect to a cluster node.
+<Notice type="tip">
+Teleport issues certificates that are good from a few hours to minutes before they auto-expire without any action.
+The shorter the duration for these certificates, the better.
+Ideally, certs should be issued only for the duration of a session.
+In practice, several hours or the duration of the workday are OK too.
+The expiry date in certificates can not be forged
+without invalidating the certificates, so any system can validate the certificate.
+</Notice>
 
-### Issuing Node certificates
+### X.509 certificates
 
-Node Certificates identify a node within a cluster and establish the permissions
-of the node to access other Teleport services. The presence of a signed
-certificate on a node makes it a cluster member.
+X.509 certificates are the same certificates you use when accessing websites with a browser. They bind
```

Comment:
We should standardise on either `x.509` or `X.509` throughout all docs. They're both used interchangably here.

Reply:
we should go for X.509